src/css/components/_c-hero.scss

```diff
@@ -50,17 +50,26 @@
 	.template-team &,
 	.template-contribute &,
 	.template-spotlight &,
-	.template-404 & {
+	.template-404 &,
+	.template-collections & {
```

Comment:
Thank you for working with our absolutely wonk layout styling. This is mostly on me 😓